package.json

```diff
@@ -28,6 +28,11 @@
   "devDependencies": {
     "@commitlint/cli": "^16.2.3",
     "@commitlint/config-conventional": "^16.2.1",
+    "eslint": "^8.12.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-unicorn": "^41.0.1",
```

Comment:
Don't think these are needed here.

Reply:
I was getting missing package errors when running `eslint` from the root dir. These packages were a result of me incrementally adding plugins until eslint stopped complaining. 🤷🏼

Reply:
Oh weird. What NPM version are you running? 8?

Reply:
I was running ^8

Reply:
Hm, were you thinking b/c they're installed by the `packages/eslint` that we'd be able to run this from the root and resolve those plugins?

Reply:
Yeah I thought since we're running the `lint` script wit h Lerna that it'd take the local executable from those packages. Leaving these deps here is cool tho, I was just confused.